fixtures/webpack.config.js

```diff
@@ -28,11 +28,7 @@ module.exports = {
   plugins: [
     new CleanWebpackPlugin(),
     new I18nModular({ emitFile }),
-    new ManifestPlugin(),
-    new CompressionPlugin({
```

Comment:
compression not needed anymore?

Reply:
no and we also remove it from our builds. the point here was to test an example config, it doesn't really matter if we test compression plugin as well. but maybe I add it again just to see that jsons are not being gzipped, that would be annoying

Reply:
added it back